nucleus/src/main/java/za/co/absa/subatomic/infrastructure/team/TeamAutomationHandler.java

```diff
@@ -59,30 +58,64 @@ public void createNewTeam(TeamEntity newTeamEntity) {
             teamSlackIdentity = new TeamSlackIdentity(
                     newTeamEntity.getSlackDetails().getTeamChannel());
         }
+
         TeamCreated teamCreated = new TeamCreated(newTeamEntity.getTeamId(),
                 newTeamEntity.getName(), newTeamEntity.getDescription(),
                 new TeamMemberId(newTeamEntity.getCreatedBy().getMemberId()),
                 teamSlackIdentity);
 
+        AtomistMemberBase atomistMemberBase = new AtomistMemberBase(
+                teamMemberEntity.getFirstName(),
+                teamMemberEntity.getDomainUsername(),
+                teamMemberSlackIdentity);
+
         TeamCreatedWithDetails newTeam = new TeamCreatedWithDetails(teamCreated,
-                new AtomistMemberBase(
-                        teamMemberEntity.getFirstName(),
-                        teamMemberEntity.getDomainUsername(),
-                        teamMemberSlackIdentity));
+                atomistMemberBase);
 
         ResponseEntity<String> response = restTemplate.postForEntity(
                 atomistConfigurationProperties.getTeamCreatedEventUrl(),
                 newTeam,
                 String.class);
 
         if (response.getStatusCode().is2xxSuccessful()) {
-            log.info("Atomist has ingested event successfully: {} -> {}",
+            log.info("Atomist has ingested TeamCreatedEvent successfully: {} -> {}",
                     response.getHeaders(), response.getBody());
         }
     }
 
+    public void teamSlackChannelCreated(TeamEntity teamEntity, TeamMemberEntity teamMemberEntity) {
+        // Raise event for TeamSetupCompletedEvent
+
+        AtomistTeam atomistTeam = new AtomistTeamMapper().createAtomistTeam(teamEntity);
+
+        TeamMemberSlackIdentity teamMemberSlackIdentity = new TeamMemberSlackIdentity(
+                teamMemberEntity.getSlackDetails().getScreenName(),
+                teamMemberEntity.getSlackDetails().getUserId());
+
+        AtomistMemberBase atomistMemberBase = new AtomistMemberBase(
+                teamMemberEntity.getFirstName(),
+                teamMemberEntity.getDomainUsername(),
+                teamMemberSlackIdentity);
+
+        TeamSlackChannelCreated ingestableObject = new TeamSlackChannelCreated(
```

Comment:
Please rename this type to reflect the name of the new event. TeamSetupCompleted I think?

Reply:
correct an oversight on my part, thanks